src/edu/csus/ecs/pc2/ui/AnswerClarificationPane.java

```diff
@@ -176,7 +177,7 @@ private JPanel getButtonPane() {
 
     private ClarificationAnswer getClarificationFromFields() {
         ClarificationAnswer clarificationAnswer = new ClarificationAnswer(getAnswerTextArea().getText(), getContest().getClientId(), 
-                getSendToAllCheckBox().isSelected(), getContest().getContestTime());
+                getSendToAllCheckBox().isSelected(), new ElementId[0],getContest().getContestTime());
```

Comment:
What is ElementId[0] ?  Comment needed or something.

Reply:
It should be null or a new constructor is needed.

Reply:
Constructor added so new array is not needed to initialize.